From 77472800a6ce146257c05c9f88a21e7f82a2b3fe Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Blinkov=20=60rifux=60=20Vladimir=20=E2=99=AA?=
 <contact@rifux.dev>
Subject: [PATCH 1/3] socrates: Project Matrixx initial support [Vanilla]
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

PUNCTUATION_SPACE for MATRIXX_MAINTAINER & MATRIXX_CHIPSET was used to fix Matrixx info in "About Device"

Example of how is maintainer name looks without PUNCTUATION_SPACE:
"by Vladimir >> out/target/product/socrates/system/build.prop; echo Blinkov"

Signed-off-by: Blinkov `rifux` Vladimir ♪ <contact@rifux.dev>
---
 lineage_socrates.mk | 27 ++++++++++++++++++++++++++-
 1 file changed, 26 insertions(+), 1 deletion(-)

diff --git a/lineage_socrates.mk b/lineage_socrates.mk
index 8b7ef3f..92e6399 100644
--- a/lineage_socrates.mk
+++ b/lineage_socrates.mk
@@ -1,5 +1,5 @@
 #
-# Copyright (C) 2024 The LineageOS Project
+# Copyright (C) 2024 The LineageOS Project, Vladimir `rifux` Blinkov
 #
 # SPDX-License-Identifier: Apache-2.0
 #
@@ -14,6 +14,7 @@ $(call inherit-product, device/xiaomi/socrates/device.mk)
 # Inherit LineageOS configurations
 $(call inherit-product, vendor/lineage/config/common_full_phone.mk)
 
+# AOSP flags
 PRODUCT_NAME := lineage_socrates
 PRODUCT_DEVICE := socrates
 PRODUCT_MODEL := Redmi K60 Pro
@@ -21,3 +22,27 @@ PRODUCT_BRAND := Redmi
 PRODUCT_MANUFACTURER := Xiaomi
 
 BUILD_FINGERPRINT := Redmi/socrates/socrates:13/UKQ1.230804.001/V816.0.11.0.UMKCNXM:user/release-keys
+
+# Matrixx flags
+PRODUCT_SYSTEM_NAME := socrates
+PRODUCT_SYSTEM_DEVICE := socrates
+PRODUCT_BUILD_PROP_OVERRIDES += \
+    PRIVATE_BUILD_DESC="socrates-user 13 UKQ1.230804.001 V816.0.11.0.UMKCNXM release-keys" \
+    TARGET_DEVICE=$(PRODUCT_SYSTEM_DEVICE) \
+    TARGET_PRODUCT=$(PRODUCT_SYSTEM_NAME)
+
+TARGET_BOOT_ANIMATION_RES := 1440
+TARGET_HAS_UDFPS := true
+
+MATRIXX_BUILD_TYPE := Unofficial
+MATRIXX_MAINTAINER := "Vladimir Blinkov"
+MATRIXX_CHIPSET := "Snapdragon 8 Gen 2"
+MATRIXX_BATTERY := 5000mAh
+MATRIXX_DISPLAY := 1440x3200
+
+# Matrixx build flags
+# | Google
+#WITH_GMS := true
+
+# | Vanilla
+WITH_GMS := false
\ No newline at end of file
-- 
2.47.0

